language/stdlib/modules/libra_system.move

```diff
@@ -72,6 +72,16 @@ module LibraSystem {
         });
     }
 
+    public fun grant_reconfiguration_capability(): ReconfigurationCapability {
+         // Only the Association can add new validators
+         Transaction::assert(Transaction::sender() == 0xA550C18, 1);
+         ReconfigurationCapability {}
+    }
+
+    public fun reconfigure_with_capability(capability: &ReconfigurationCapability) acquires ValidatorSet {
```

Comment:
Is this used? Should we kill it?

Reply:
I meant to leave it here to use it for on chain config in the next PR.